feeder/src/vote.ts

```diff
@@ -315,13 +318,32 @@ export async function vote(args: VoteArgs): Promise<void> {
         console.error(err.message)
       }
 
+      console.log('vote: lcd client unavailable, rotating to next lcd client.')
+      rotateLCD(client, currentLCDIndex, maxLCDIndex, args)
```

Comment:
Seems this rotate is not properly working because, currentLCDIndex is const and it is passed by value. You can return new LCDIndex in rotateLCD or you can pass it as reference as described in [here](https://stackoverflow.com/a/35608020)

Reply:
currentLCDIndex should be non-const